test/MyLiquidityBloc_test.dart

```diff
@@ -0,0 +1,256 @@
+import 'package:ax_dapp/pages/pool/MyLiqudity/bloc/MyLiquidityBloc.dart';
+import 'package:ax_dapp/pages/pool/MyLiqudity/models/MyLiquidityItemInfo.dart';
+import 'package:ax_dapp/repositories/usecases/GetAllLiquidityInfoUseCase.dart';
+import 'package:ax_dapp/service/Controller/usecases/GetWalletAddressUseCase.dart';
+import 'package:fpdart/fpdart.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import 'MyLiquidityBloc_test.mocks.dart';
+
+@GenerateMocks([GetAllLiquidityInfoUseCase, GetWalletAddressUseCase])
+void main() {
+  var mockRepoUseCase = MockGetAllLiquidityInfoUseCase();
+  var mockWalletAddressUseCase = MockGetWalletAddressUseCase();
+  var myLiquidityBloc = MyLiquidityBloc(
+      repo: mockRepoUseCase, controller: mockWalletAddressUseCase);
+  final testWalletAddress = "testWalletAddress";
+  final List<MyLiquidityItemInfo> testMyLiquidityItemInfo = [
```

Comment:
this is overkill, your logic doesn't even use this data, mock an object or objects and put them in a list.